README.md

```diff
@@ -454,6 +454,14 @@ Bundle succeeded and output file to index.js
 $ node .
 ```
 
+Since `spago bundle` also requires the source project to be built, it will do so before
+bundling. Due to this, we can specify the source paths, and the pass-through arguments to the
+`purs` compiler.
+
+```bash
+$ spago bundle --main Main --to index.js --path 'another_source/**/*.purs' -- -o myOutput/
```

Comment:
I'd say we could skip this text: the reason is that if someone needs to pass these flags they will do it already when calling `build`, and they'll want to do it anyways (so they don't need this text); the rest of the people (those not passing the "build" flags) don't care already about them, so they don't need this text (as all of this information is anyways there when you do `spago bundle -h`)

I'd say we could just mention something like "`spago bundle` will build your project if it hasn't been built already"

Reply:
Ok, I removed these and added a section on how to skip the build